### PR TITLE
Update bridges subtree

### PR DESCRIPTION
**Pull request by serban300**
Update bridges subtree

### CODE REVIEW

File: bridges/snowbridge/parachain/pallets/ethereum-client/Cargo.toml [hunk 1]
```diff
@@ -38,7 +38,7 @@ snowbridge-core = { path = "../../primitives/core", default-features = false }
 snowbridge-ethereum = { path = "../../primitives/ethereum", default-features = false }
 primitives = { package = "snowbridge-beacon-primitives", path = "../../primitives/beacon", default-features = false }
 static_assertions = { version = "1.1.0", default-features = false }
-bp-runtime = { path = "../../../../../bridges/primitives/runtime", default-features = false }
+bp-runtime = { path = "../../../../primitives/runtime", default-features = false }
```
Comment (svyatonik) [inline bridges/snowbridge/parachain/pallets/ethereum-client/Cargo.toml:41]: (this change won't beak anything in Snowbridge repo @claravanstaden ?)

Comment (svyatonik) [reply bridges/snowbridge/parachain/pallets/ethereum-client/Cargo.toml:41]: (also see similar changes below)

**Review by svyatonik — APPROVED** [review]
Don't know if we need to add the `prdoc` for that - we have removed `type LaneMessageVerifier` from messages pallet configuration, so we may add that to the release notes.

File: bridges/snowbridge/parachain/pallets/ethereum-client/Cargo.toml (hunk 1 above)
Comment (serban300) [reply bridges/snowbridge/parachain/pallets/ethereum-client/Cargo.toml:41]: It shouldn't. In the end it's the same path. Just optimized.

**Comment by serban300** [comment]
> Don't know if we need to add the `prdoc` for that - we have removed `type LaneMessageVerifier` from messages pallet configuration, so we may add that to the release notes.

I'm not sure if we should do this since the bridges are not used in production yet.

File: bridges/snowbridge/parachain/pallets/ethereum-client/Cargo.toml (hunk 1 above)
Comment (svyatonik) [reply bridges/snowbridge/parachain/pallets/ethereum-client/Cargo.toml:41]: I agree, but just wanted to double check :)

**Comment by paritytech-cicd-pr** [comment]
The CI pipeline was cancelled due to failure one of the required jobs.
**Job name**: cargo-clippy
**Logs**: https://gitlab.parity.io/parity/mirrors/polkadot-sdk/-/jobs/4968711

**Review by bkontur — APPROVED** [review]
lgmt

File: bridges/snowbridge/parachain/primitives/router/Cargo.toml [hunk 2]
```diff
@@ -28,7 +28,7 @@ xcm = { package = "staging-xcm", path = "../../../../../polkadot/xcm", default-f
 xcm-builder = { package = "staging-xcm-builder", path = "../../../../../polkadot/xcm/xcm-builder", default-features = false }
 xcm-executor = { package = "staging-xcm-executor", path = "../../../../../polkadot/xcm/xcm-executor", default-features = false }
 
-snowbridge-core = { path = "../../primitives/core", default-features = false }
```
Comment (claravanstaden) [inline bridges/snowbridge/parachain/primitives/router/Cargo.toml:31]: Nice catch 😅

**Review by claravanstaden — APPROVED** [review]

**Pull request merged**